flow/envs/bay_bridge/base.py

```diff
@@ -209,12 +209,15 @@ def apply_toll_bridge_control(self):
 
     # TODO: decide on a good reward function
     def compute_reward(self, state, rl_actions, **kwargs):
+        """See class definition."""
         return np.mean(self.vehicles.get_speed(self.vehicles.get_ids()))
 
     """ The below methods need to be updated by child classes. """
 
     def _apply_rl_actions(self, rl_actions):
+        """Update by child classes."""
```

Comment:
Should this read "update using child classes" or "implemented by child classes"?